src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
+        mpAboutDlg = new dlgAboutDialog( this );
```

Comment:
It is interesting to run Mudlet through GammaRay and see all the normally hidden things hanging around that are not deleted when not in used {mind you, the duplicate instances that leak resources are more of a concern in that area, e.g. toolbars \**sigh*\*}